### PR TITLE
version 2.2.1

### PR DESCRIPTION
**Pull request by xiazeyu**
Hey, it's me again xD.
Here's the update below:
```
Improve coding quality
Add ImmediateMode (Issue #27)
Translated README into Chinese
```
and some questions:
`Does Version in config file mean the version it was created? Maybe we can use it to mention the user to have a look at new configs.`

and, known bugs:
`Known BUG: Error when you put the program into a path that contains Chinese character.`
I think it is not much important, because Chinese character will cause many issue when developing even in other languages.

and last, please build the version and release it for me.
Thanks to the Chinese network, I cannot use glide because it will give a lot of error of downloading dependencies, so I can only build it manually, I don't know whether it will causes problems.

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+> Node.js Portable 现在被集成进了 [Neard](http://neard.io) ！
```
Comment (crazy-max) [inline README_CN.md]: Can you rename the Chinese README to the standard naming ?
See https://github.com/tiimgreen/github-cheat-sheet for more info.
I think here we have to name the README : `README.zh-cn.md`

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@
   <a href="https://flattr.com/submit/auto?user_id=crazymax&url=https://github.com/crazy-max/nodejs-portable"><img src="https://img.shields.io/badge/flattr-this-green.svg?style=flat-square" alt="Flattr this!"></a>
 </p>
 
+[Chinese 中文](README_CN.md)
```
Comment (crazy-max) [inline README.md]: After renaming the file replace with this ? :

In **README.md** :

*Read this in other languages: [English](README.md), [简体中文](README.zh-cn.md).*

In **README.zh-cn.md** (translate "Read this in other languages" to chinese) :

*Read this in other languages: [English](README.md), [简体中文](README.zh-cn.md).*

**Comment by xiazeyu** [comment]
@crazy-max I've done the changes, have a look.xD

**Comment by crazy-max** [comment]
Nice!

**Comment by crazy-max** [comment]
@xiazeyu And for your question :

> Does `Version` in config file mean the version it was created?
> Maybe we can use it to mention the user to have a look at new configs.

Version is informative for the user and when a new var is detected in a new release, this one is merged in the actual config.

**Comment by xiazeyu** [comment]
@crazy-max but I found that the Version in the `nodejs-portable.conf` doesn't update with the `nodejs-portable.exe`.
I read the code, and really found that `when a new var is detected in a new release, this one is merged in the actual config.` is a ingenious idea, but I still don't know what does `Version` in `nodejs-portable.conf` means:

Version just be created at first-start, and is it only for user to know what version of program created this conf file?
None of the code use `Version` in `nodejs-portable.conf`, and it won't be updated when you update `nodejs-portable.exe`. User can just look into the title of the shell to find out the version.

**Comment by crazy-max** [comment]
@xiazeyu Hum you right i missed the implementation i have made on [WindowsSpyBlocker ](https://github.com/crazy-max/WindowsSpyBlocker) 😄

**Comment by xiazeyu** [comment]
er, sorry that I don't understand what you said.
I mean when you replace the main program with a newer version, the 'version' in the config file **won't be updated**, unless you delete the config and let the program to create a completed new config.
In conclusion, I think 'version' in the config file makes no sense.

**Comment by xiazeyu** [comment]
so, you may just remove it.

**Pull request merged**